website/templates/website/event_create.html

```diff
@@ -0,0 +1,22 @@
+{% extends "website/default.html" %}
+
+{% block title %} {{ block.super }} - Create Event {% endblock %}
+
+{% block content %}
+  {{ block.super }}
+  <div class="container">
+    <h1>Create event</h1>
+
+    <form method="post" novalidate>
```

Comment:
How come we're using novalidate?

Reply:
are we using server-side validation? If so we need to disable browser's default validation mechanism using `novalidate` :open_mouth: